README.md

```diff
@@ -162,3 +162,216 @@ ARGS:
     <pattern>    the search pattern, a regular expression (optional)
     <path>       the root directory for the filesystem search (optional)
 ```
+
+## Examples
+
+First to get `fd`'s help run: 
+
+```
+fd --help
+```
+
+Let's assume we have some files we need to search through like so: 
+
+```
+fd_examples
+├── .gitignore
+├── not_file
+├── sub_dir
+│   ├── .here_be_tests
+│   ├── more_dir
+│   │   ├── .not_here
+│   │   ├── even_further_down
+│   │   │   ├── not_me.sh
+│   │   │   ├── test_seven
+│   │   │   └── testing_eight
+│   │   ├── not_file -> /Users/fd_user/fd_examples/not_file
+│   │   └── test_file_six
+│   ├── test_file_five
+│   ├── test_file_four
+│   └── test_file_three
+├── test_file_one
+├── test_file_two
+├── test_one
+└── this_is_a_test
+```
+
+Let's do a recursive search for anything that has the name test in it (`fd` will start in the current directory and follow symbolic links by default):
```

Comment:
Minor correction: by default, `fd` does *not* follow symlinks. I think we can simply remove the "and follow symbolic links by default" part here.